### PR TITLE
Yuning_PR_resnet50_AWS

### PR DESCRIPTION
**Pull request by YuningQiu**
Edit the run scripts for running on AWS instance.

### CODE REVIEW

File: closed/Intel/code/resnet50/pytorch-cpu/run_offline_accuracy.sh [hunk 1]
```diff
@@ -30,30 +30,30 @@ if [ -z "${RN50_FULL}" ]; then
...
+APP=${PWD}/build/bin/mlperf_runner
 
 export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${CONDA_PREFIX}/lib
 
 if [ -e mlperf_log_accuracy.json ]; then
     rm mlperf_log_accuracy.json
 fi
 
-numactl -C 0-55,56-111 -m 0,1 ${APP} --scenario Offline \
+numactl -C 0-47 -m 0 ${APP} --scenario Offline \
```
Comment (louie-tsai) [inline closed/Intel/code/resnet50/pytorch-cpu/run_offline_accuracy.sh:56]: one quick question, why do we have a fix number 47 here?

File: closed/Intel/code/resnet50/pytorch-cpu/run_server.sh [hunk 2]
```diff
@@ -29,30 +29,33 @@ if [ -z "${RN50_FULL}" ]; then
...
+APP=${PWD}/build/bin/mlperf_runner
 
 export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${CONDA_PREFIX}/lib
 
 if [ -e "mlperf_log_summary.txt" ]; then
     rm mlperf_log_summary.txt
 fi
 
-numactl -C 0-55,56-111 -m 0,1 ${APP} --scenario Server  \
+numactl -C 0-47 -m 0,1 ${APP} --scenario Server  \
```
Comment (louie-tsai) [inline closed/Intel/code/resnet50/pytorch-cpu/run_server.sh:58]: same here. why we have fix number 47 here?

File: closed/Intel/code/resnet50/pytorch-cpu/run_server_accuracy.sh [hunk 3]
```diff
@@ -29,30 +29,33 @@ if [ -z "${RN50_FULL}" ]; then
...
+APP=${PWD}/build/bin/mlperf_runner
 
 export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${CONDA_PREFIX}/lib
 
 if [ -e mlperf_log_accuracy.json ]; then
     rm mlperf_log_accuracy.json
 fi
 
-numactl -C 0-55,56-111 -m 0,1 ${APP} --scenario Server \
+numactl -C 0-47 -m 0 ${APP} --scenario Server \
```
Comment (louie-tsai) [inline closed/Intel/code/resnet50/pytorch-cpu/run_server_accuracy.sh:58]: same here.

**Review by louie-tsai — COMMENTED** [review]
got a quick question for fixed numactl number

**Comment by louie-tsai** [comment]
@YuningQiu 
I will merge the PR even there is still a question there because I plan to merge all AWS materials into one branch.
thanks

File: closed/Intel/code/resnet50/pytorch-cpu/run_offline_accuracy.sh (hunk 1 above)
Comment (YuningQiu) [reply closed/Intel/code/resnet50/pytorch-cpu/run_offline_accuracy.sh:56]: MicrosoftTeams-image
I met this issue saying that "55 and 56-111" is invalid. After checking lscpu, I change the number to "0-47"

**Pull request merged**